probe-rs/targets/STM32F4 Series.yaml

```diff
@@ -424,248 +421,247 @@ variants:
           sector_size: 528
           page_size: 528
           erased_byte_value: 255
-  - name: STM32F427ZITx
+  - name: STM32F407IGTx
     memory_map:
       - Ram:
           range:
             start: 536870912
-            end: 537067520
+            end: 537001984
           is_boot_memory: false
       - Flash:
           range:
             start: 134217728
-            end: 136314880
+            end: 135266304
           is_boot_memory: true
           sector_size: 528
           page_size: 528
           erased_byte_value: 255
-  - name: STM32F479AIYx
+  - name: STM32F407VETx
     memory_map:
       - Ram:
           range:
             start: 536870912
-            end: 537198592
+            end: 537001984
           is_boot_memory: false
       - Flash:
           range:
             start: 134217728
-            end: 136314880
+            end: 134742016
           is_boot_memory: true
-          sector_size: 65536
-          page_size: 1024
+          sector_size: 528
+          page_size: 528
           erased_byte_value: 255
-  - name: STM32F479VGTx
+  - name: STM32F407VGTx
     memory_map:
       - Ram:
           range:
             start: 536870912
-            end: 537198592
+            end: 537001984
           is_boot_memory: false
       - Flash:
           range:
             start: 134217728
             end: 135266304
           is_boot_memory: true
-          sector_size: 65536
-          page_size: 1024
+          sector_size: 528
+          page_size: 528
           erased_byte_value: 255
-  - name: STM32F412ZGJx
-    part: 0x411
```

Comment:
I think this got removed as well, please add it to the sorted file as well.

Reply:
patch set updated with all manufacturer and part id